pipelines/prod-manual-release-pipeline.yml

```diff
@@ -1017,12 +1017,11 @@ jobs:
     input_mapping: {release: census-rm-kubernetes-release}
     output_mapping: {unpacked-release: census-rm-kubernetes-release-unpacked}
   - task: "Run Helm"
-    file: census-rm-deploy/tasks/prod-helm.yml
+    file: census-rm-deploy/tasks/rabbitmq-provision.yml
```

Comment:
Gavin we've addressed this issue within the ./setup-rabbitmq.sh script which now discerns whether production configuration should be used depending on whether you want rabbit node pools. This will keep Whitelodge with dev-like configuration - whilst saddling greylodge, blacklodge, performance and the prods with prod-like configuration.